scrapers/USA/KY/index.js

```diff
@@ -0,0 +1,51 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+const scraper = {
+  country: 'USA',
+  state: 'KY',
+  priority: 1,
```

Comment:
Go ahead and add yourself as the maintainer if you want your name on the sources page and are interested in keeping this scraper up to date! Do it like this: https://github.com/lazd/coronadatascraper/blob/master/scrapers/BRA/index.js#L15-L24